### PR TITLE
Consume ONNX 1.13.1 in ONNX Runtime

### PR DESCRIPTION
**Pull request by jcwchen**
### Description

Consume ONNX 1.13.1 in ONNX Runtime. (ONNX 1.13.0 to ONNX 1.13.1)

### Motivation and Context

ONNX 1.13.1 patch was just released yesterday. This PR is making ORT's ONNX submodule consistent with the latest released ONNX. Not sure whether this PR is really needed, but let me make it ready. Previous PR for testing ONNX 1.13.1rc2 : https://github.com/microsoft/onnxruntime/pull/14634.

Fixed [AB#13174](https://aiinfra.visualstudio.com/6a833879-cd9b-44a4-a9de-adc2d818f13c/_workitems/edit/13174) .

### CODE REVIEW

**Review by thiagocrepaldi — DISMISSED** [review]
LGTM

Yes, the ONNX 1.13.1 is a nice and possibly harmless change. It contains a fix (typo) to Col2Im unit tests

**Review by snnn — COMMENTED** [review]
Please run https://github.com/microsoft/onnxruntime/blob/main/cgmanifests/generate_cgmanifest.py

**Comment by jcwchen** [comment]
> Please run https://github.com/microsoft/onnxruntime/blob/main/cgmanifests/generate_cgmanifest.py

Thanks for catching this! I have run it and updated `cgmanifest.json`. PTAL.

**Comment by snnn** [comment]
The ARM test failed in "ONNX Runtime Python Test Pipeline" is fixed in the latest main branch code, but not in your branch. It's ok to ignore it here.

**Review by snnn — APPROVED** [review]

File: cgmanifests/generated/cgmanifest.json [hunk 1]
```diff
@@ -302,7 +302,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "1ba785612a79fe749aa1e478336e534743372639",
+          "commitHash": "ad834eb73ee0cd9b6fa9ea892caeed5fa17d7dc0",
```
Comment (snnn) [inline cgmanifests/generated/cgmanifest.json:305]: ad834eb73ee0cd9b6fa9ea892caeed5fa17d7dc0 is the right one.

**Comment by snnn** [comment]
> If this is not strictly required for the ORT patch let's skip it.

You mean the 1.14.1 patch release?

**Comment by pranavsharma** [comment]
> > If this is not strictly required for the ORT patch let's skip it.
> 
> You mean the 1.14.1 patch release?

yes

**Review by pranavsharma — APPROVED** [review]

**Pull request merged**